src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -16,6 +16,12 @@
 import threading
 import types  # pylint: disable=import-error
 
+if sys.version_info >= (3, 0):
```

Comment:
Funny thing. #24626 contains a better way to import `unquote`

```suggestion
from six.moves.urllib.parse import unquote  # pylint: disable=import-error
```
But I wouldn't change it in Mimic only.